examples/wasm/docker-compose-wasm.yaml

```diff
@@ -0,0 +1,23 @@
+version: "3.7"
+
+services:
+
+  wasm_compile_update:
+    image: envoyproxy/envoy-build-ubuntu:b480535e8423b5fd7c102fd30c92f4785519e33a
```

Comment:
its not ideal hardcoding the (commit head sha) like this, im wondering if we can add a `latest` tag  on this dockerhub repo

Reply:
afaict there is no intention to resolve this - so afaict the only way to document this is hardcoding it - i think anything else is outside the scope of this PR